### PR TITLE
Update integrate-office-365-ti-with-mde.md

### LINKED ISSUE #12109: This feature is deprecated -> I received the following answer from microsoft support:
**Issue:** The Office 365 Threat Intelligence connection toggle in the Advanced features page under the M365 Defender > Settings > Endpoints is stuck in a Pending state.  
**Cause:** This setting was used when MDO and MDE were in different portals back in the time. After the convergence of security experiences into unified portal that we call M365 Defender today, these settings are irrelevant and they don't have any functionality associated with them.
**User impact:** None
**Current Status:** Customers can safely ignore the status of the control until it is removed from the portal.

Please update documentation with an info panel containing above information.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 3681d1c8-2fad-ddd7-4948-40e59ea1b0bd
* Version Independent ID: 910578e6-7b25-a577-dfe7-8538236649ec
* Content: [Use Microsoft Defender for Office 365 together with Microsoft Defender for Endpoint](https://learn.microsoft.com/en-us/microsoft-365/security/office-365-security/integrate-office-365-ti-with-mde?view=o365-worldwide#to-integrate-microsoft-defender-for-office-365-with-microsoft-defender-for-endpoint)
* Content Source: [microsoft-365/security/office-365-security/integrate-office-365-ti-with-mde.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/integrate-office-365-ti-with-mde.md)
* Service: **microsoft-365-security**
* Sub-service: **mdo**
* GitHub Login: @denisebmsft
* Microsoft Alias: **deniseb**

### PR DESCRIPTION
**Pull request by msbemba**
added an important note about MDO and MDE integration

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12109

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/office-365-security/integrate-office-365-ti-with-mde.md [hunk 1]
```diff
@@ -50,6 +50,9 @@ In this example, you can see that the recipients of the detected email message h
 
 ## To integrate Microsoft Defender for Office 365 with Microsoft Defender for Endpoint
 
+> [!IMPORTANT]
+> This setting was used when Microsoft Defender for Office 365 and Microsoft Defender for Endpoint were in different portals back in the time. After the convergence of security experiences into unified portal that we call M365 Defender today, these settings are irrelevant and they don't have any functionality associated with them. Customers can safely ignore the status of the control until it is removed from the portal.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/integrate-office-365-ti-with-mde.md]:
```suggestion
> This setting was used when Microsoft Defender for Office 365 and Microsoft Defender for Endpoint were in different portals previously. After the convergence of security experiences into a unified portal that is now called Microsoft 365 Defender, these settings are irrelevant and don't have any functionality associated with them. You can safely ignore the status of the control until it is removed from the portal.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@denisebmsft Please merge this PR. Thanks

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**